src/decoration/decoration.ts

```diff
@@ -0,0 +1,33 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as fs from 'fs';
+import { getNotes } from '../note-db';
+
+const decorationType = vscode.window.createTextEditorDecorationType({
+    backgroundColor: new vscode.ThemeColor('codeAnnotaion.mainColor')
```

Comment:
Ideally, the `codeAnnotation.mainColor` would come from the configuration `configuration.ts`, this way we only have one place that will manipulate the configuration, making it easier to maintain.